setuptools/command/easy_install.py

```diff
@@ -1112,8 +1110,9 @@ def install_wheel(self, wheel_path, tmpdir):
             self.execute(
                 wheel.install_as_egg,
                 (destination,),
-                ("Installing %s to %s")
-                % (os.path.basename(wheel_path), os.path.dirname(destination)),
+                (
+                    f"Installing {os.path.basename(wheel_path)} to {os.path.dirname(destination)}"
+                ),
```

Comment:
I think those parenthesis are redundant and lead to longer line

```suggestion
                f"Installing {os.path.basename(wheel_path)} to {os.path.dirname(destination)}",
```